From https://gcc.gnu.org/gcc-13/porting_to.html

Some C++ Standard Library headers have been changed to no longer
include other headers that were being used internally by the library.
As such, C++ programs that used standard library components without
including the right headers will no longer compile.

The following headers are used less widely in libstdc++ and may need
to be included explicitly when compiling with GCC 13:

    <string> (for std::string, std::to_string, std::stoi etc.)
    <system_error> (for std::error_code, std::error_category, std::system_error).
    <cstdint> (for std::int8_t, std::int32_t etc.)
    <cstdio> (for std::printf, std::fopen etc.)
    <cstdlib> (for std::strtol, std::malloc etc.)

-

Matteo Bernardini <ponce@slackbuilds.org>

diff -Naur wxsvg-1.5.24.orig/src/cairo/SVGCanvasCairo.cpp wxsvg-1.5.24/src/cairo/SVGCanvasCairo.cpp
--- wxsvg-1.5.24.orig/src/cairo/SVGCanvasCairo.cpp	2022-08-31 16:56:26.000000000 +0200
+++ wxsvg-1.5.24/src/cairo/SVGCanvasCairo.cpp	2023-04-27 22:23:05.069420000 +0200
@@ -14,6 +14,7 @@
 #include "SVGCanvasImageCairo.h"
 #include <wx/log.h>
 #include <wx/file.h>
+#include <cstdint>
 
 wxSVGCanvasCairo::~wxSVGCanvasCairo() {
 	Destroy();
